### PR TITLE
Texture prev

### PR DESCRIPTION
**Pull request by RodPaDev**
This is mostly working, the roofs needs to be done since one category can mean several pictures

### CODE REVIEW

**Comment by Heleor** [comment]
When you make UI changes in general it'll be helpful to show a screenshot of what it looks like.

File: web/maps.html [hunk 1]
```diff
@@ -167,17 +167,17 @@
                         </div>
                         <div id='tools-detail-buildings-floors' class='hidden'>
                             Floor: <select id='tools-detail-buildings-floor-list'></select>
-                            <input type='button' value='...'>
+                            <input type='button' value='...' onclick="SCENERY_EDITOR.openTexturePreviewModal('tools-detail-buildings-floor-list')">
```
Comment (Heleor) [inline web/maps.html:170]: Not what I meant. I meant a selection modal that shows all the textures available in a list and lets the user choose which one they want, just like the model view.

Comment (Heleor) [reply web/maps.html:170]: Something like this:

https://imgur.com/a/oKPvmLB

**Review by Heleor — CHANGES_REQUESTED** [review]

File: web/maps.html (hunk 1 above)
Comment (RodPaDev) [reply web/maps.html:170]: Gotcha

**Pull request closed without merging**